src/main/java/com/lofri/catchtable/domain/restaurant/controller/RestaurantController.java

```diff
@@ -19,8 +28,29 @@ public ResponseTemplate<GetRestaurantsResponse> getRestaurants(@RequestParam(nam
                                                                    @RequestParam(name = "min_price", defaultValue = "0") Integer minPrice,
                                                                    @RequestParam(name = "max_price", defaultValue = "400000") Integer maxPrice,
                                                                    @RequestParam(name = "table_type", required = false) List<String> tableType,
-                                                                   @RequestParam(name = "amenity", required = false) List<String> amenities) {
-        return null;
+                                                                   @RequestParam(name = "amenity", required = false) List<String> amenities,
+                                                                   @RequestParam(name = "page", defaultValue = "0") int pageNo,
+                                                                   @RequestParam(name = "page_size", defaultValue = "20") int pageSize,
+                                                                   @RequestHeader(name = "userid") Long userId // Todo Principle 로 변경
```

Comment:
+ Argument Resolver까지 사용해서 제가 생각하는 이상향은

```suggestion
    User user
```